lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -92,7 +92,7 @@ defmodule Phoenix.Endpoint.CowboyHandler do
   def start_link(scheme, endpoint, config, {m, f, a}) do
     case apply(m, f, a) do
       {:ok, pid} ->
-        Logger.info info(scheme, endpoint, config)
+        Logger.info info(scheme, endpoint, hd(a))
```

Comment:
what is `hd(a)` in this context? Let's name it here or pattern match it as a variable so it's clear